cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
```

Comment:
I wasn't sure either, but it probably does make sense to use the thing we have. Who knows when we'll have time to switch.